src/renderer/components/ConnectionRow.tsx

```diff
@@ -78,15 +79,17 @@ const ConnectionRow: React.FC<ConnectionRowProps> = ({
         ipcRenderer.send(SAVE_RECORD, dupe);
         break;
       case 'copy_port':
-        // eslint-disable-next-line no-case-declarations
-        const parsed = port?.match(/\d+(?![^:]*:)/g);
-        clipboard.writeText(parsed?.length ? parsed[0] : '');
+        clipboard.writeText(port);
```

Comment:
Not port anymore I am guessing?

Reply:
1. yes
2. i tried that but it looked bad. but sure
3. sure we can do that. no icon then?

Reply:
no it is the whole address now. but it still says copy port. not sure what you want to do here.